frb_dart/lib/src/basic.dart

```diff
@@ -234,6 +234,9 @@ class FrbAnyhowException implements FrbException {
   final String anyhow;
 
   FrbAnyhowException(this.anyhow);
+
+  @override
+  String toString() => anyhow;
```

Comment:
Maybe the following toString is a bit more aligned with flutter (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#override-tostring):

```suggestion
  String toString() => 'FrbAnyhowException($anyhow)';
```